source/utils/case.ts

```diff
@@ -18,3 +18,5 @@ export const toTitleCase = (str: string) => {
 		.replace(/_/g, " ")
 		.replace(/\w+/g, (match) => match.charAt(0).toUpperCase() + match.slice(1));
 };
+
+export const kababToSnakeCase = (str: string) => str.replace(/-/g, () => "_");
```

Comment:
Actually, this is not really "SnakeCase"
The name is more: "toValidVariableName"

Reply:
If I have "Hello-World" snakeCase should output: "hello_world" not "Hello_World"
because snakeCase is in lower letters.

That's why the function name is misleading

Reply:
i changed it to `kababToConstantCase`